includes/emicdora.inc

```diff
@@ -26,10 +27,33 @@ function collationtools_get_edited_collation($document_id) {
       'version2' => $_GET['version2'],
       'diff_kind' => $_GET['diff_kind'],
   )));
-  $contents = file_get_contents($url);
-  $spans = preg_split( '#</span>#', $contents, -1, PREG_SPLIT_DELIM_CAPTURE | PREG_SPLIT_NO_EMPTY);
-  $fixed_content = collationtools_fix_calliope($spans, $_GET['diff_kind']);
-
+  $raw_contents = file_get_contents($url);
+  // The MVD engine breaks br's where we don't want them broken.
+  // Replacing with tokens makes it all work.
+  $contents = str_replace($br_token, '<br />', $raw_contents);
+  $dom = new DOMDocument();
+  $dom->loadHTML($contents);
+  $bodies = $dom->getElementsByTagName('body');
+  $body = $bodies->item(0);
+  // Get rid of comment nodes.
+  foreach ($dom->childNodes as $node) {
+    if ($node->nodeType === XML_COMMENT_NODE) {
+      $node->parentNode->removeChild($node);
+    }
+  }
+  $all_nodes = $body->childNodes;
+  foreach ($all_nodes as $node) {
+    // Convert text nodes to spans.
+    if ($node->nodeType == XML_TEXT_NODE) {
```

Comment:
Not a huge deal: Should probably be consistent with the operator... `==` vs `===`. (`===` is used for comments above)